### ISSUE TITLE
run error

### ISSUE THREAD

**Issue by KnewHow** [ssloy/tinyrenderer#71]
I read your markdown, then execute command in my Ubuntu, but get a error: "texture file ../obj/floor_spec.tga loading can't open file ../obj/floor_spec.tga failed". I review your master code in github, I find in "obj" folder, the "floor_spec.tga" is not exist, please add it into github, let's  view whole effective. Thank your!

**Comment by ssloy** [comment]
This is not an error, just a warning saying that the floor does not have specular texture, the code runs just fine without it.

**Comment by KnewHow** [comment]
Yes, you are right! I not read all article of this. I very thank you teaching that!
